Configurations/15-ios.conf

```diff
@@ -11,7 +11,7 @@ my %targets = (
         template         => 1,
         inherit_from     => [ "darwin-common" ],
         sys_id           => "iOS",
-        disable          => [ "shared", "async" ],
+        disable          => [ "async" ],
```

Comment:
IOS is an unadopted platform.  As such CI doesn't test this and I (personally) have no way to verify it.  Can you offer some evidence that this works properly?

Reply:
I did try it locally and it produces a dylib. Are there other tests I can do? Would it be possible to add a ci job to test ios builds?

Reply:
the change looks good. one possible test worth to try is to check if  'make install' drops the dynamic library to install prefix. there is not much else we can do I think.

Reply:
I did a test as requested and it works as described.